ios/AllAboutOlaf/Info.plist

```diff
@@ -76,7 +76,7 @@
 	<key>NSCalendarsUsageDescription</key>
 	<string>We use your calendar to add events to your calendar so that you remember what you wanted to attend.</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>We only use your location when you would expect us to, and never when you aren&apos;t using the app.</string>
+	<string>We only use your location when you would expect us to, and never when you aren't using the app.</string>
```

Comment:
Please revert this (Xcode changes it, and something else changes it back… it's a struggle)